dlc_developer_config.toml

```diff
@@ -32,12 +32,12 @@ ec2_tests = true
 
 ### Off by default
 efa_tests = false
```

Comment:
This can remain `false` - there are no EFA tests for HF afaik. The EFA failures we saw in a prior release were related to an unrelated bug that we identified and fixed.